### PR TITLE
Fix react depencendy

### PR DESCRIPTION
**Pull request by konupek**
In order to make Phoenix Components compatible with latest version of create-react app, I had to update couple of dependencies:

- React (min.: 17.0.0)
- react-php (min version 5.3.2, since v4 is not compatible with React 17)

### CODE REVIEW

**Comment by FilipPyrek** [comment]
Cauko @konupek 

1. can you please check your CircleCI account? Just sign-up with your GitHub account. The tests should be executed.. but they are not. So I assume there is some permissions issue.
2. we use conventional commits here and I guess because of mistake #162 husky didn't get installed for you, thus the commit lint was not executed. Could you please do a rebase so that the commits are compliant with the commit lint?

Thanks 🙇‍♂️ 💜

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@purple/phoenix-components",
-	"version": "4.4.0",
+	"version": "4.4.1",
```
Comment (FilipPyrek) [inline package.json:3]: Did you update the version manually or was it automated? Since we use standard-version, it automatically (when executed) changes version of package based on commits since previous release. So if you did that manually, you can revert this change.

File: package.json [hunk 2]
```diff
@@ -52,11 +52,11 @@
 		"nanoid": "^3.1.23",
 		"react-dropzone": "10.2.0",
 		"react-inlinesvg": "^2.3.0",
-		"react-pdf": "^4.1.0",
+		"react-pdf": "^5.3.2",
```
Comment (samuelg0rd0n) [inline package.json:55]: Why did you update this package? Did you check any potential compatibility issues?

File: package.json [hunk 3]
```diff
@@ -52,11 +52,11 @@
...
 		"react-select": "^4.3.1"
 	},
 	"peerDependencies": {
-		"react": ">=16.2",
+		"react": "^17.0.2",
```
Comment (samuelg0rd0n) [inline package.json:59]: We need Phoenix Components to be compatible also with React 16 because that's what is currently used in MyAxiory. Please try something like `>=16.2 <= 17` or `^16.2.0 || ^17.0.0`
